js/components/graph/__tests__/Sidebar.test.js

```diff
@@ -53,14 +53,6 @@ describe("Sidebar", () => {
     expect(container.getByText("FCE Count: 0.5")).toBeDefined();
   });
 
-  it("Clicking the reset button should reset the FCE Count to 0.0", async () => {
```

Comment:
I miscommunicated with the team. We are planning to move the reset selection button into a new dialogue rather than add it as a graph button, so I reverted the changes so as to avoid unnecessary changes.